### PR TITLE
Update blackbox.c

### PR DESCRIPTION
**Pull request by AndreySemjonov**
Need separate TARGET field in BB headers

### CODE REVIEW

File: src/main/blackbox/blackbox.c [hunk 1]
```diff
@@ -1090,6 +1090,7 @@ static bool blackboxWriteSysinfo(void) {
     switch (xmitState.headerIndex) {
         BLACKBOX_PRINT_HEADER_LINE("Firmware type", "%s",                   "Cleanflight");
         BLACKBOX_PRINT_HEADER_LINE("Firmware revision", "%s %s (%s) %s",    FC_FIRMWARE_NAME, "3.7.0", shortGitRevision, targetName);
```
Comment (Quick-Flash) [inline src/main/blackbox/blackbox.c:1092]: will this "tricking" of BBL ever end? are we to the point that we can display the correct firmware version and use the emu bbl instead?

**Review by Quick-Flash — APPROVED** [review]

File: src/main/blackbox/blackbox.c (hunk 1 above)
Comment (AndreySemjonov) [reply src/main/blackbox/blackbox.c:1092]: What do you mean nu tricking? I made already that we can show correct version :) I need target to separate IMuF for helio and non helio in headers

Comment (Quick-Flash) [reply src/main/blackbox/blackbox.c:1092]: "3.7.0" we are telling it that we are using version 0.3.7 which is clearly shown here 
image

Comment (AndreySemjonov) [reply src/main/blackbox/blackbox.c:1092]: https://github.com/emuflight/EmuFlight/pull/624
This need to be merged. @nerdCopter want to wait before EBB completed completely

**Pull request merged**